pages/pipelines/hosted_agents/code_access.md

```diff
@@ -1,14 +1,15 @@
 # Hosted agents code access
 
-This page details changes that need to be made to the GitHub integration to provide code access to the Buildkite hosted agents.
+Natively, Buildkite hosted agents can only be used with GitHub integrated repositories. To utilize a private repository from another provider, we have [made a plugin](https://github.com/buildkite-plugins/git-ssh-checkout-buildkite-plugin) available that is documented below.
 
 To learn more about changes that may need to be completed at an individual pipeline level, see [Pipeline migration](/docs/pipelines/hosted-agents/pipeline-migration).
 
-## Private repositories
+## GitHub private repositories
 
-During the trial, access to private repositories is only supported through GitHub. If you have requirements to access private repositories from another source control service, please contact support.
+To use a private GitHub repository with Buildkite hosted agents, you will need to authorize Buildkite to access your repository.
```

Comment:
I'm starting to use this convention ... mention closer to the start that the "Buildkite organization administrator" role is required for this and link through to the relevant documentation on this...

```suggestion
To use a private GitHub repository with Buildkite hosted agents, you need to authorize Buildkite to access your repository. This process can only be conducted by [Buildkite organization administrators](/docs/team-management/permissions#manage-teams-and-permissions-organization-level-permissions)
```

I'd avoid using callouts/admonitions to indicate this information as based on the number of times we'd need to mention this in the docs, the number of callouts would become far too distracting.